examples/wait_pod_running.js

```diff
@@ -0,0 +1,40 @@
+import { Kubernetes } from 'k6/x/kubernetes';
+
+let pod = {
+    apiVersion: "v1",
+    kind:       "Pod",
+    metadata: {
+        name:      "busybox",
+        namespace:  "default"
+    },
+    spec: {
+        containers: [
+            {
+                name:    "busybox",
+                image:   "busybox",
+                command: ["sh", "-c", "sleep 30"]
+            }
+        ]
+    }
+}
+
+export default function () {
+  const kubernetes = new Kubernetes();
+
+  // create namespace with random name with prefix 'test-'
+  const ns = kubernetes.helpers().createRandomNamespace("test-")
```

Comment:
Script needs to be updated for latest changes. The `createRandomNamespace` method is no longer valid.

Reply:
Good catch. Fixed